__tests__/_/binaries.js

```diff
@@ -8,7 +10,7 @@ const zlib = require('zlib');
 
 
 export const p = os.platform();
-export const bv = (binaries_version || version).split('.')[0];
+export const bv = process.env.BINARIES_VERSION ? process.env.BINARIES_VERSION.replace(/\./g, '_') : (binaries_version || version).split('.')[0];
```

Comment:
There is no need to normalize passed variable here: we should do these checks at the stage we upload binaries using this value.
Just use its value "as is".